test/axes/timeAxisTests.ts

```diff
@@ -149,6 +149,54 @@ describe("TimeAxis", () => {
     svg.remove();
   });
 
+  it("tick labels do not overlap with tick marks in top orientation", () => {
+    let svg = TestMethods.generateSVG(400, 100);
+    scale.domain([new Date("2010-01-01"), new Date("2014-01-01")]);
+    axis = new Plottable.Axes.Time(scale, "top");
```

Comment:
This line might be confusing. Let's declare a new variable called topAxis (and prefix it with `let`)